code/modules/atmospherics/machinery/atmospherics.dm

```diff
@@ -317,7 +324,7 @@ Pipelines + Other Objects -> Pipe network
 			user.forceMove(target_move.loc) //handles entering and so on
 			user.visible_message("You hear something squeezing through the ducts.", "You climb out of the ventilation system.")
 		else if(target_move.can_crawl_through())
-			if(returnPipenet(target_move) != target_move.returnPipenet())
```

Comment:
This didn't work because one would return null, and the other would return the parent. Thus always being different from each other